extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/KubernetesAndWorkerTaskRunner.java

```diff
@@ -232,6 +233,16 @@ public Optional<InputStream> streamTaskLog(String taskid, long offset) throws IO
     return Optional.absent();
   }
 
+  @Override
+  public TaskLocation getTaskLocation(String taskId)
+  {
+    TaskLocation taskLocation = kubernetesTaskRunner.getTaskLocation(taskId);
```

Comment:
What's the desired behavior if `TaskLocation taskLocation = kubernetesTaskRunner.getTaskLocation(taskId);` throws exception?

Shoudn't we fallback to workerTaskRunner ?

Reply:
i feel it really probably shouldn't show an exception b/c this is the logic
    final KubernetesWorkItem workItem = tasks.get(taskId);
    if (workItem == null) {
      return TaskLocation.unknown();
    } else {
      return workItem.getLocation();
    }

Reply:
i feel like its fine to propagate an exception up b/c something is probably v wrong if it does throw one

Reply:
Fair enough